source/octf/proto/statistics.proto

```diff
@@ -74,30 +69,35 @@ message IoStatisticsEntry {
      */
     StatisticsEntryValues size = 1;
 
+    /**
+     * Number of IO events
+     */
+    uint64 count = 2;
+
```

Comment:
Add this field as field no. 3 ( one of the reserved fields.) The reserved fields are for extending the existing data structure - so this exact use case. Leave other field numbers intact